Chassis/.gitignore

```diff
@@ -1 +1,2 @@
-Chassis-backups
+# Backups
+Chassis-backups/*
```

Comment:
How about the lock file?

Reply:
I put that into the KiCAD-Library gitignore

Reply:
You right, I blind